### DISCUSSION TITLE
Undoing the deleted ones;  Linking to mail.

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#897] [Help & FAQ] [answered]
How can I undo my deleteS ?  :
Getting back the deleted habits(if it happens, unfortunately!!).

How can I link the present using account to my using mail, and having an another account for another mail, HOW(Like Tasks app) ?

**Comment by hiqua** [comment, marked as answer]
I don't think you can undelete, as the text mentions clearly ("cannot be undone"). If it's archived you can unarchive it though. If you have a backup, you could try to restore it of course. There might be some backups in the app storage, but I'm not sure, and anyway they're quickly replaced.

About your second question, I guess you'd like to have different "accounts" with different sets of habit? Why do you want that?
